src/System.Text.Json/tests/Utf8JsonWriterTests.cs

```diff
@@ -3747,6 +3747,13 @@ public void WriteNumbers(bool formatted, bool skipValidation, string keyString)
             doubles[2] = double.MinValue;
             doubles[3] = 12.345e1;
             doubles[4] = -123.45e1;
+            doubles[5] = 6.022e-23;
+            doubles[6] = -6.022e-23;
+            doubles[7] = 1e200;
+            doubles[8] = -1e200;
+            doubles[9] = 1e-200;
+            doubles[10] = -1e-200;
```

Comment:
While these may be reasonable tests, these aren't testing the code change.

To test the code change you'd need to add a test like

```C#
[Theory]
[InlineData("6.022e-23", false)]
[InlineData("6.022e-23", true)]
[InlineData("6.022e23", false)]
[InlineData("6.022e23", true)]
[InlineData("6.022e+23", false)]
[InlineData("6.022e+23", true)]
[InlineData("-6.022e-23", false)]
[InlineData("-6.022e-23", true)]
[InlineData("-6.022e23", false)]
[InlineData("-6.022e23", true)]
[InlineData("-6.022e+23", false)]
[InlineData("-6.022e+23", true)]
public static void WriteNumberDecimalScientific(string value, bool indented)
{
    WriteSimpleValue(indented, value);
}
```

In JsonElementWriteTests.cs, around line 39 (between WriteNumberScientific and WriteNumberOverprecise).

Reply:
roger that I didn't get it right

Reply:
Please remove these extra inputs, as this is an unrelated change.  (These numbers were parsed by the compiler, not the JSON writer, they aren't evaluated, just written down via Utf8Formatter... because they can't be "invalid")